server/src/main/java/timely/store/DataStoreImpl.java

```diff
@@ -215,6 +196,33 @@ public void run() {
         }
     }
 
+    private static final EnumSet<IteratorScope> AGEOFF_SCOPES = EnumSet.allOf(IteratorScope.class);
+
+    private void removeAgeOffIterators(Connector con, String tableName) throws Exception {
+        Map<String, EnumSet<IteratorScope>> iters = con.tableOperations().listIterators(tableName);
+        iters.forEach((name, scopes) -> {
+            if (name.startsWith("ageoff")) {
+                try {
+                    con.tableOperations().removeIterator(tableName, name, AGEOFF_SCOPES);
+                } catch (Exception e) {
+                    // do nothing
```

Comment:
Wouldn't an exception here potentially leave the table in an inconsistent state?